src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -221,20 +221,20 @@ public void connectionAdded(final INode to) {
             if (portString == null || portString.trim().length() <= 0) {
               portString = "3300";
             }
-            final String message = "Your computer is not reachable from the internet.\r\n"
-                + "Please make sure your Firewall allows incoming connections (hosting) for TripleA.\r\n"
-                + "(The firewall exception must be updated every time a new version of TripleA comes out.)\r\n"
+            final String message = "Your computer is not reachable from the internet.\n"
+                + "Please make sure your Firewall allows incoming connections (hosting) for TripleA.\n"
+                + "(The firewall exception must be updated every time a new version of TripleA comes out.)\n"
                 + "And that your Router is configured to send TCP traffic on port " + portString
                 + " to your local ip address.\r\n"
```

Comment:
Sorry, missed this in the review of #2665.  Should the CR be removed from this line, as well?